ci/jenkins/README.md

```diff
@@ -0,0 +1,50 @@
+# Theia CI: Jenkins
+
+## Reasons for Jenkins
+
+We have tests as Github Actions but Jenkins allows tests running on a cluster of
+multiple nodes and offers better environment setup options.
+
+## List of Jobs
+
+| Job Name | Description                                    | Trigger Phase   | 
```

Comment:
Found some markdown format error when I debug that dead link by cherry-picking your PR, please fix these:
```
ci/jenkins/README.md:10:80 MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]
ci/jenkins/README.md:11:80 MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]
ci/jenkins/README.md:14 MD012/no-multiple-blanks Multiple consecutive blank lines [Expected: 1; Actual: 2]
ci/jenkins/README.md:15 MD012/no-multiple-blanks Multiple consecutive blank lines [Expected: 1; Actual: 3]
ci/jenkins/README.md:47 MD012/no-multiple-blanks Multiple consecutive blank lines [Expected: 1; Actual: 2]
```